pinot-query-planner/src/test/resources/queries/WindowFunctionPlans.json

```diff
@@ -646,6 +803,21 @@
           "\n"
         ]
       },
+      {
+        "description": "single OVER(PARTITION BY) row_number with select col and group by",
+        "sql": "EXPLAIN PLAN FOR SELECT a.col1, ROW_NUMBER() OVER(PARTITION BY a.col1) FROM a GROUP BY a.col1, a.col3",
```

Comment:
May be I missed it but I don't see any tests for OVER(PARTITION BY .. ORDER BY ..) variant ? Is that not supported in this PR ?

Reply:
nvm... I just saw them. Ignore